WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Blaze/DashboardBlazeCampaignCardCell.swift

```diff
@@ -8,12 +8,22 @@ final class DashboardBlazeCampaignCardCell: DashboardCollectionViewCell {
     private lazy var contentStackView: UIStackView = {
         let stackView = UIStackView()
         stackView.axis = .vertical
-        stackView.layoutMargins = Constants.contentInsets
-        stackView.spacing = 4
-        stackView.isLayoutMarginsRelativeArrangement = true
         return stackView
     }()
 
+    private lazy var createCampaignButton: UIButton = {
+        let button = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.contentHorizontalAlignment = .leading
+        button.setTitle(Strings.createCampaignButton, for: .normal)
+        button.addTarget(self, action: #selector(buttonCreateCampaignTapped), for: .touchUpInside)
+        button.titleLabel?.font = WPStyleGuide.fontForTextStyle(.callout, fontWeight: .semibold)
```

Comment:
Looks like it should be bold according to Figma specs (1Osrn9gb2xLaKG6lIft3WC-fi-1510_19803)
```suggestion
        button.titleLabel?.font = WPStyleGuide.fontForTextStyle(.callout, fontWeight: .bold)
```